pkg/kubefed2/options/options.go

```diff
@@ -71,3 +74,28 @@ func (o *CommonSubcommandOptions) SetName(args []string) error {
 	o.ClusterName = args[0]
 	return nil
 }
+
+type FederationConfigOptions struct {
+	ClusterNamespace string
+}
+
+func GetOptionsFromFederationConfig(hostConfig *rest.Config, namespace string) (bool, string, error) {
+	client, err := genericclient.New(hostConfig)
+	if err != nil {
+		glog.V(2).Infof("Failed to get federation clientset: %v", err)
```

Comment:
Should the error be wrapped with this message and returned, rather than logged?  Otherwise the error will be logged redundantly (and with no context).

Reply:
done.